### PR TITLE
Added FR-Minecraft.net

### PR DESCRIPTION
**Pull request by fantomitechno**
*A website solo !*

So I added FR-Minecraft.net a french website of news on minecraft 

> Proof it work : 

image

image

image

image

image

image

image

Forum part :

image

image

image

image

image

image

and that's it

### CODE REVIEW

**Review by Bas950 — APPROVED** [review]

File: websites/F/FR-Minecraft/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "author": {
+    "name": "fantomitechno 🦊",
+    "id": "563749920683720709"
+  },
+  "url": "fr-minecraft.net",
+  "description": {
+    "en": "FR-Minecraft : News Minecraft, tips, tools et all the help necessary to play Minecraft !",
+    "fr": "FR-Minecraft : L'actualité Minecraft, des astuces, des outils et toute l'aide necessaire pour jouer à Minecraft !."
```
Comment (ririxidev) [inline websites/F/FR-Minecraft/dist/metadata.json:10]: Please remove ``FR-Minecraft :`` from description

**Review by ririxidev — CHANGES_REQUESTED** [review]
You should take a look at this:

File: websites/F/FR-Minecraft/dist/metadata.json (hunk 1 above)
Comment (Bas950) [reply websites/F/FR-Minecraft/dist/metadata.json:10]: also remove the space before the ! at the end.

Comment (fantomitechno) [reply websites/F/FR-Minecraft/dist/metadata.json:10]: Forget this I change it

**Comment by fantomitechno** [comment]
Modified

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**